open-api/rest-catalog-open-api.yaml

```diff
@@ -248,6 +248,7 @@ paths:
         The server might also add properties, such as `last_modified_time` etc.
       operationId: createNamespace
       requestBody:
+        required: true
```

Comment:
can you also please add the same for `getToken`?

Reply:
Done